projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -70,8 +74,17 @@ export class TabParagraphContainerComponent
     )
   );
 
-  select(tabNum: number): void {
-    this.activeTabNum = tabNum;
+  select(tabNum: number, event): void {
+    this.tabSubscription = this.breakpointService
```

Comment:
Can you confirm it doesn't introduce memory leaks?
We create new subscription each time the `select` method is called. But we don't dispose the previous subscription that was stored in `this.tabSubscription`. So the subscription holders are lost!